### PR TITLE
Update Faqs section

### PR DESCRIPTION
**Pull request by apanjain**
add answers and refactor css

### CODE REVIEW

File: frontend/src/routes/Queries/Answer.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        })
+    }
+    componentWillReceiveProps(nextProps){
+        this.setState({
+            visibility : nextProps.visibility
+        })
+    }
+    render() { 
+        return ( 
+            <React.Fragment>
```
Comment (hacksdump) [inline frontend/src/routes/Queries/Answer.js]: Just a suggestion. You can use <></> tags for fragments.

File: frontend/src/routes/Queries/questions.js [hunk 2]
```diff
@@ -4,26 +4,26 @@ import Button from '../../components/Button';
...
 ]
 
 class Questions extends Component {
     state = {  }
     render() { 
         return ( 
         <div className={styles.questions}>
-            <div className={styles.questionsColumn1}>{questionText1.map((data,i) => <QuestionsBox key={i} text={data}/>)}</div>
-            <div className={styles.questionsColumn2}>{questionText2.map((data,i) => <QuestionsBox key={i+4} text={data}/>)}</div>
+            <div className={styles.questionsColumn1}>{questionText1.map((data,i) => <QuestionsBox key={i} query={data}/>)}</div>
```
Comment (hacksdump) [inline frontend/src/routes/Queries/questions.js]: Create array for questions and use arr.map((data, index) => <Component props>) to generate elements.

File: frontend/src/routes/Queries/questions.module.scss [hunk 3]
```diff
@@ -5,14 +5,22 @@
...
     display: inline-grid;
     margin: 3.815625rem auto 1.875rem 1.8125rem;
   }  
 }
 
 .questionsColumn1{
+  display:block;
+  width:90%;
+  margin: auto;
   @media screen and (min-width: 1180px){
```
Comment (hacksdump) [inline frontend/src/routes/Queries/questions.module.scss]: create a new variable for this breakpoint.

File: frontend/src/routes/Queries/QuestionsBox.module.scss [hunk 4]
```diff
@@ -1,59 +1,73 @@
...
+}
+.up{
+  -webkit-transform: scaleY(-1);
+  -moz-transform: scaleY(-1);
+  -o-transform: scaleY(-1);
+   transform: scaleY(-1);
+}
+
+//Answer
+.answer{
```
Comment (hacksdump) [inline frontend/src/routes/Queries/QuestionsBox.module.scss]: Answers won't render well in mobile view.

File: frontend/src/routes/Queries/questions.js (hunk 2 above)
Comment (apanjain) [reply frontend/src/routes/Queries/questions.js]: its an array already

File: frontend/src/routes/Queries/Answer.js (hunk 1 above)
Comment (apanjain) [reply frontend/src/routes/Queries/Answer.js]: okay

File: frontend/src/routes/Mentors/Mentors.module.scss [hunk 5]
```diff
@@ -5,7 +5,15 @@
     background-image: linear-gradient($light-blue, $white);
     color: $grey-level-0_7;
     // padding: 5%;
+<<<<<<< HEAD
```
Comment (hacksdump) [inline frontend/src/routes/Mentors/Mentors.module.scss]: Clean this

Comment (hacksdump) [reply frontend/src/routes/Mentors/Mentors.module.scss]: Please resolve the conflicts.

File: frontend/src/routes/Mentors/Mentors.module.scss [hunk 6]
```diff
@@ -75,7 +99,15 @@
...
     width:249px;
+<<<<<<< HEAD
+<<<<<<< HEAD
+    max-width: 90%;
+=======
+    max-width: 90vw;
+>>>>>>> mentors page mobile view
+=======
     max-width: 90%;
+>>>>>>> requested changes
```
Comment (hacksdump) [inline frontend/src/routes/Mentors/Mentors.module.scss]: more conflicts

Comment (hacksdump) [reply frontend/src/routes/Mentors/Mentors.module.scss]: shouldn't have ended up in this PR.

File: frontend/src/routes/Queries/questions.module.scss (hunk 3 above)
Comment (apanjain) [reply frontend/src/routes/Queries/questions.module.scss]: done

**Pull request merged**